tests/phpunit/includes/factory.php

```diff
@@ -94,6 +94,35 @@ class WP_UnitTest_Factory_For_Attachment extends WP_UnitTest_Factory_For_Post {
 	function create_object( $file, $parent = 0, $args = array() ) {
 		return wp_insert_attachment( $args, $file, $parent );
 	}
+
+	function create_upload_object( $file, $parent = 0 ) {
```

Comment:
Adding this factory method is something wonderboymusic and I discussed after https://core.trac.wordpress.org/changeset/33807, which could replace redundant `_make_attachment` helper functions  in `tests/image/intermediate_size.php`, `tests/post/attachment.php`, and `tests/ajax/MediaElement.php`. I still need to create a separate ticket for that.